keps/sig-scheduling/3902-decoupled-taint-manager/README.md

```diff
@@ -140,24 +140,27 @@ The creation and startup of the default taint-manager is performed by `kube-cont
 Kubernetes already has a good coverage of node `No-Execute` eviction. We will add tests only for the changed code.
 
 #####  Unit tests
-- `pkg/controller/taintmanager`: 2023-06-03 - 0% (to add)
+- `pkg/controller/tainteviction`: 2024-09-18 - 81.8% 
   - Enable and disable the new `TaintEvictionController`
   - The new `TaintEvictionController` acts on node taints properly
-- `pkg/controller/apis/config/v1alpha1`: 2023-06-03 - 0%
-  - Test new configuration
-- `pkg/controller/nodelifecycle`: 2023-06-03 - 74.8%
+- `pkg/controller/nodelifecycle`: 2024-09-18 - 68%
   - Test the combined controller
-- `pkg/controller/nodelifecycle/scheduler`: 2023-06-03 - 90%
+- `pkg/controller/nodelifecycle/scheduler`: 2024-09-18 - 85.3%
```

Comment:
This is a 5% decrease. Also worth bringing it up.

Reply:
will try